index.html

```diff
@@ -77,110 +118,118 @@
     </div>
 </div>
 
-<div class="container spaced" id="post-list">
-    <div class="row">
-        <div class="col-12">
-            <h2 class="header">Recent posts</h2>
+<h2 class="left_offset" style="text-align: center;">Things we're working on ...</h2>
```

Comment:
Could this be in the same formatting as the `"Getting more done after 2am..." <h2> tag` or vice versa, just consistent

Reply:
:+1:

Reply:
Addressed in _sass/_typography.scss lines 36-45 of the diff. Changes applied to dev site.